src/extensions/nodejsMigrationHelper.ts

```diff
@@ -0,0 +1,75 @@
+import { FirebaseError } from "../error";
+import * as extensionsApi from "./extensionsApi";
+import { logPrefix } from "./extensionsHelper";
+import { promptOnce } from "../prompt";
+import * as utils from "../utils";
+
+const changeMsg =
+  "Node.js 8 has been deprecated. It’s recommended to update this extension to ensure it is running on Node.js 10.";
```

Comment:
For updates: This feels weird to me as a warning in the beginning of the update process because it's telling you to go do something but you're already doing it. @kevinthecheung wondering if it makes sense to combine with the billingMsgUpdate?

Reply:
Maybe we don't need `displayNodejsChangeNotice()`?

Reply:
Removed!

Reply:
LGTM! I think we're all set as long as Annie is good.